github/password_token_yaratish.md

```diff
@@ -0,0 +1,49 @@
+#### Githubdagi password token qismini yaratish
+
+* `github` sahifamizga kiramiz. Foydalanuvchining sozlamalar (settings) qismiga boramiz.
+
+<p align="center">
+    <img src="./images/foydalanuvchi_sozlamalar_qismi.png" width="450">
+</p>
+
+* `Developer settings` qismiga boramiz.
+
+<p align="center">
+    <img src="./images/developer_settings.png" width="600">
+</p>
+
+* `Personal access tokens` qismga borib `Tokens (classic)` qismini tanlaymiz.
+
+<p align="center">
+    <img src="./images/token_classic.png" width="600">
+</p>
+
+*  `Generate new token (classic)` qismiga boramiz.
+<p align="center">
+    <img src="./images/generate_new_token.png" width="600">
+</p>
+
+* `Note` qismiga ixtiyoriy parol kiritamiz, masalan `ochiqai123`. Pastida ko'rsatilgan kichik to'rtburchak shaklidagi ko'rsatmalarning hammasini belgilab chiqamiz.
+
+<p align="center">
+    <img src="./images/new_personal_token.png" width="600">
+</p>
+
+* `Generate token` tugmasini bosamiz.
+
+<p align="center">
+    <img src="./images/generate_token.png" width="600">
+</p>
+
+* Yaratib o'tgan yangi tokenimizni ko'rishingiz mumkin.
+<br>
+`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`
+
+<p align="center">
+    <img src="./images/yangi_token.png" width="600">
+</p>
+
+* Bu tokendan nusxa olib kerakli joyga saqlab qo'yishimiz lozim.
+Foydalanuvchi uchun githubdagi parolni talab qilsa foydalanamiz. 
+<br>
+Masalan `Password for 'https://Nuriddin-Dev@github.com':`
```

Comment:
Bunday qilib o'zgartisak yaxshi bo'ladi:

* Yaratib o'tgan yangi tokenimizni ko'rishingiz mumkin.
   * `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`

<p align="center">
    <img src="./images/yangi_token.png" width="600">
</p>

* Bu tokendan nusxa olib kerakli joyga saqlab qo'yishimiz lozim.
Foydalanuvchi uchun githubdagi parolni talab qilsa foydalanamiz. Masalan:
  
  ```shell
  `Password for 'https://Nuriddin-Dev@github.com':`
  ```

Reply:
to'g'irladim